CONTRIBUTING.rst

```diff
@@ -29,13 +29,28 @@ We use flake8 to check some PEP8 errors/warnings. This will be checked with
 Simple Bandwidth Scanner is moving towards 100% test coverage. Please help us
 reach that goal, or at least not move us away from it.
 
+Git and GitHub Guidelines
+=========================
+
 **Write good commit messages** that at least follow the spirit of
 :ref:`this example <commit-msg>`.
 
-**Write many small commits** each containing an atomic change instead of one
-large mega-commit. This not only makes code review easier, but it also makes
-commits that show up in ``git blame`` 10 years from now make more sense.
+Strive to **write many small commits** each containing an atomic change instead
```

Comment:
However, small commits makes not clear what is a complete feature.
In other projects, any commit should pass all tests, and if adding a new feature, it should contain also the test. Following that, commits can be actually quite big.
So far i never heard of a workflow that would prioritize ``git blame``.  If a project is mature/in production, developers try to modify code making as less changes as possible to other parts of the code (probably made by other developers).